docs/subnets/deploying-subnets-on-prod.md

```diff
@@ -0,0 +1,189 @@
+# Deploying Subnets on Production Infrastructure
+
+## Introduction
+
+After architecting your Subnet environment on the [local machine](create-a-local-subnet.md), proving the design and testing it out on [the testnet](create-a-fuji-subnet.md), eventually you will need to deploy your Subnet to production environment. Running a subnet in production is much more involved than local and testnet deploys, as your Subnet will have to take care of real world usage, maintaining uptime, upgrades and all of that in a potentially adversarial environment. The purpose of this document is to point out a set of general considerations and propose potential solutions to them.
```

Comment:
```suggestion
After architecting your Subnet environment on the [local machine](create-a-local-subnet.md), proving the design and testing it out on [the testnet](create-a-fuji-subnet.md), eventually you will need to deploy your Subnet to production environment. Running a Subnet in production is much more involved than local and testnet deploys, as your Subnet will have to take care of real world usage, maintaining uptime, upgrades and all of that in a potentially adversarial environment. The purpose of this document is to point out a set of general considerations and propose potential solutions to them.
```
Please search and replace all `subnet` to `Subnet`

Reply:
I capitalized Subnet when talking about a particular subnet (your Subnet). Left with lowercase when talking about generic subnets. If the style guide is to capitalize subnets wholesale, I can correct.

Reply:
Marketing said to capitalize subnets